spec/features/results_spec.rb

```diff
@@ -25,6 +25,8 @@
       include: [:subjects],
     )
 
+    allow(Settings).to receive_message_chain(:google, :maps_api_key).and_return("alohomora")
+    allow(Settings).to receive_message_chain(:google, :maps_api_url).and_return("https://maps.googleapis.com/maps/api/staticmap")
```

Comment:
don't think you need to mock it out

Reply:
We wanted to mock it out as it's not the thing we wanted to test here - in the future the location for the key or url might change. I can get rid of it though, if you think it's more appropriate.